### PR TITLE
Add integrations list

### PR DESCRIPTION
**Pull request by jadb**
Refs https://github.com/thephpleague/oauth2-server/commit/c174b6fc65f034dd6547da6ab7bb429a20988eb0

### CODE REVIEW

**Comment by colinodell** [comment]
I have added this project to the wiki.  Thanks!

**Comment by jadb** [comment]
@colinodell I am afraid not everyone goes to the wiki and other league packages are showing integrations in the README, wouldn't it be better to follow same template (look) for all the league projects? 

Thanks!

**Comment by colinodell** [comment]
There's a link in the README to the relevant wiki page: https://github.com/thephpleague/commonmark/blob/master/README.md#community-extensions  Perhaps this section should be revised to make it more prominent/clear?

I'd also be open to listing them on the project website (see the [`gh-pages` branch](https://github.com/thephpleague/commonmark/tree/gh-pages)).

I'd prefer these approaches over maintaining the list within the README itself.

**Comment by jadb** [comment]
I've been giving this a little more thought and I still think that if all the League packages followed the same structure for the README would be much better for end users like myself. 

I mean those packages are all using the same doc templates (gh-pages) and in general, a very standardized way of doing things, why would integrations for this particular package be listed in a different place than all the rest? The other packages might end up moving to wiki, I don't know, but standardizing it so people know where to look for all the league packages is what am trying to get to.

Maybe others can share their opinions too? /cc @philsturgeon @frankdejonge

**Comment by colinodell** [comment]
The [skeleton](https://github.com/thephpleague/skeleton) template we use doesn't have an integrations section, and only some projects include this section, so it's definitely not standardized.  You're right though, providing a similar experience will be best for the users.

I think my biggest concern is that README is versioned.  It's highly possible that any "supported" integration/extension listed there may not actually support new versions we release.  OTOH, this probably isn't a big deal, and perhaps README inclusion could encourage those package authors to stay up-to-date or risk removal?

You make some good points though so I'll reconsider this and get back to you soon.  I'm open to any additional feedback you or others may have.

**Comment by GrahamCampbell** [comment]
Missing https://github.com/GrahamCampbell/Laravel-Markdown for Laravel. :)

**Comment by jadb** [comment]
:+1: for also including the Laravel one, want me to add to this PR @GrahamCampbell ?

**Comment by GrahamCampbell** [comment]
:+1:

**Comment by philsturgeon** [comment]
I like it being in the README.

File: README.md [hunk 1]
```diff
@@ -148,6 +148,11 @@ This package is **not** part of CommonMark, but rather a compatible derivative.
 
 Documentation can be found at [commonmark.thephpleague.com][docs].
 
+## Integrations
+
+- [CakePHP 3](https://github.com/gourmet/commonmark)
+- [Laravel 5](https://github.com/GrahamCampbell/Laravel-Markdown)
```
Comment (GrahamCampbell) [inline README.md]: It's also laravel 5 too.

Comment (GrahamCampbell) [inline README.md]: It's also laravel 4 too.

Comment (GrahamCampbell) [inline README.md]: https://github.com/GrahamCampbell/Laravel-Markdown/tree/v2.1.0

**Comment by colinodell** [comment]
Alright README it is! Thanks for being patient and persistent with me ;-)

**Comment by jadb** [comment]
@colinodell @philsturgeon @GrahamCampbell thanks!

**Pull request merged**